ios/brave-ios/Sources/Onboarding/WelcomeFocus/FocusSystemSettingsView.swift

```diff
@@ -0,0 +1,148 @@
+            )
+            .dynamicTypeSize(dynamicTypeRange)
+            .fixedSize(horizontal: false, vertical: true)
+            .multilineTextAlignment(.center)
+
+          Text(Strings.FocusOnboarding.defaultBrowserScreenDescription)
+            .font(
+              Font.custom("Poppins-Medium", size: 17)
+            )
+            .dynamicTypeSize(dynamicTypeRange)
+            .fixedSize(horizontal: false, vertical: true)
+            .multilineTextAlignment(.center)
+            .foregroundColor(Color(braveSystemName: .textTertiary))
+        }
+        .padding(.bottom, 24)
+
+        Spacer()
+
+        LottieAnimationView(
+          name: colorScheme == .dark ? "browser-default-dark" : "browser-default-light",
+          bundle: .module
+        )
+        .loopMode(.loop)
+        .resizable()
+        .aspectRatio(contentMode: .fit)
+        .background(Color(braveSystemName: .pageBackground))
+        .clipShape(RoundedRectangle(cornerRadius: 12.0))
+        .overlay(
+          RoundedRectangle(cornerRadius: 12.0)
+            .stroke(Color(braveSystemName: .textTertiary), lineWidth: 1)
+        )
+        .padding(.bottom, 24)
+
+        Spacer()
+
+        VStack(spacing: 24) {
+          Button(
+            action: {
+              if let settingsUrl = URL(string: UIApplication.openSettingsURLString) {
+                UIApplication.shared.open(settingsUrl)
+              }
+
+              Preferences.FocusOnboarding.urlBarIndicatorShowBeShown.value = true
+              shouldDismiss = true
+            },
+            label: {
+              (Text("\(Strings.FocusOnboarding.systemSettingsButtonTitle) ")
+                + Text(Image(systemName: "arrow.right")))
+                .font(.body.weight(.semibold))
+                .foregroundColor(Color(.white))
+                .dynamicTypeSize(dynamicTypeRange)
+                .padding()
+                .foregroundStyle(.white)
+                .frame(maxWidth: .infinity)
+                .background(Color(braveSystemName: .buttonBackground))
+            }
+          )
+          .clipShape(RoundedRectangle(cornerRadius: 12.0))
+          .overlay(RoundedRectangle(cornerRadius: 12.0).strokeBorder(Color.black.opacity(0.2)))
+
+          Button(
+            action: {
+              Preferences.FocusOnboarding.urlBarIndicatorShowBeShown.value = true
+              shouldDismiss = true
+            },
+            label: {
+              Text(Strings.FocusOnboarding.laterActionButtonTitle)
+                .font(.subheadline.weight(.semibold))
+                .foregroundColor(Color(braveSystemName: .textSecondary))
+                .dynamicTypeSize(dynamicTypeRange)
+            }
+          )
+          .background(Color.clear)
+        }
+      }
+      .padding(.vertical, shouldUseExtendedDesign ? 48 : 24)
+      .padding(.horizontal, shouldUseExtendedDesign ? 75 : 20)
+    }
+  }
+}
+
+struct FocusSystemSettingsView_Previews: PreviewProvider {
```

Comment:
wrap in `#if DEBUG`

Reply:
Fixed at  https://github.com/brave/brave-core/commit/91c6c912a8207d7f813f771f7e0e16ff5050eb1d